apps/taxes/src/pages/providers/avatax/[id]/logs.tsx

```diff
@@ -0,0 +1,74 @@
+import { Provider } from "jotai";
+import { Section } from "../../../../modules/ui/app-section";
+
+import { Text } from "@saleor/macaw-ui/next";
+import { useRouter } from "next/router";
+import { AvataxLogs } from "../../../../modules/avatax/logs/ui/avatax-logs";
+import { AppPageLayout } from "../../../../modules/ui/app-page-layout";
+import { AVATAX_LOG_LIMIT } from "../../../../modules/avatax/logs/avatax-client-logger";
+
+const LogsInstructions = () => {
+  return (
+    <Section.Description
+      title="AvaTax Logs"
+      description={
+        <>
+          <Text as="p" marginBottom={8}>
+            Taxes App records all AvaTax API calls and responses. You can use this information to
+            debug issues with AvaTax.
+          </Text>
+          <Text as="p" marginBottom={8}>
+            Only the last {AVATAX_LOG_LIMIT} logs are stored.
+          </Text>
+          <Text as="p" marginBottom={8}>
+            The naming convention for each event is: <b>[WebhookName] API method name</b>.
+          </Text>
+        </>
+      }
+    />
+  );
+};
+
+const Header = () => {
+  return <Section.Header>Display logs for your AvaTax configuration</Section.Header>;
+};
+
+const LogsAvataxPage = () => {
```

Comment:
same as other comments - its unnecessary duplication of code

you can have a single page "LogsPage" that will be mounted on `/logs/:configId`. The rest can happen automatically. trpc will return logs and type of provider for this config id

see how its done in CMS, where ~5 providers are used dynamically